docs/src/upgrade/release-3.4.x.asciidoc

```diff
@@ -177,6 +177,17 @@ when dealing with that event. To make this easier, the event now raises with a `
 
 link:https://issues.apache.org/jira/browse/TINKERPOP-1831[TINKERPOP-1831]
 
+==== Gremlin.Net: Configurable Max and Min ConnectionPool Sizes
```

Comment:
Please shorten the header a bit because it turns into a table of contents item that ends up crowding the frame. Perhaps just go with "Gremlin.Net Connection Pool" - it just needs to be long enough to catch someone's eye who is looking.

Reply:
Also, while this is a nice explanation of the this feature, perhaps it's time to add a "Configuration" section to the reference documentation itself? If no one else agrees, that's fine - just a thought I had given this new feature. No new users are going to find this nice explanation here in the upgrade docs i don't think.

Reply:
I like the idea of adding docs for the configuration of Gremlin.Net to the docs, especially since the configuration is getting more and more complicated. I'll try to come up with some docs for this.